rails.md

```diff
@@ -55,46 +60,58 @@ Answer the following questions. First, without external resources. Challenge you
 
 3b. Which of the above routes must always be passed params and why?
 
+probably update because it needs a param to update to and from
 
+4. What is the public folder used for in Rails
 
-4. What is the public folder used for in Rails?
-
-  Your answer:
+  Your answer:for when we want to post stuff to the public for them to access
 
   Researched answer:
-
-
+  In many MVC frameworks (e.g. Rails), the public folder is used to serve static files. So for example, you often see 404.
 
 5. Write a rails route for a controller called "main" and a page called "game" that takes in a parameter called "guess"
 
-
+  "/game:guess"=> "main#game"
 
 
 6. In an html form, what does the "action" attribute tell you about the form? How do you designate the HTTP verb for the form?
 
-  Your answer:
+  Your answer:C(create)R(read)U(update)D(delete) match GET POST PATCH PUT DELETE
+
+  <%=form_tag action: "/main/game", method: :get do %>
+  <% end %>
 
   Researched answer:
+ The action attribute is used to inform the browser what page to call once the "submit" button is pressed.
+ the http verbs are used when you want a certain action to happen , like DELETE/destroy
+
 
 
 
 7. Name two rails generator commands and what files they create:
 
-  Your answer:
+  Your answer: rails generate controller Main
+  rails g controller Main
+
 
   Researched answer:
 
+  rails g model Bike
+  rails g controller Bike
+
```

Comment:
If you are going to do a generate for model you would also pass the db attributes.